core/src/components/cat-datepicker/cat-datepicker.scss

```diff
@@ -1,5 +1,7 @@
 @use 'variables' as *;
 @use 'mixins' as *;
+@import '~flatpickr/dist/flatpickr.css';
```

Comment:
why are those imports needed?

Reply:
we need it because when we attach datepicker to custom dev warapper element inside shadow component (and cat-datepicker is a shadow component) then the html is encapsulated for global css. So before the datepicker modal was always inserted to the body and global css (flatpickr/dist/flatpickr.css was a part of global css) applies to calendar modal html, but it's different for html inside shadow dom and I wasn't able to find a better solution

Reply:
understood.